qiskit_nature/problems/second_quantization/molecular/molecular_problem.py

```diff
@@ -13,24 +13,17 @@
 """The Molecular Problem class."""
 from typing import List, Optional, Tuple
 
-from qiskit_nature.drivers.qmolecule import QMolecule
-from qiskit_nature.drivers import FermionicDriver
-from qiskit_nature.operators import FermionicOp
-from qiskit_nature.operators.second_quantization import SecondQuantizedOp
-from qiskit_nature.problems.second_quantization.molecular.integrals_calculators \
-    .angular_momentum_integrals_calculator import \
+from ....drivers.qmolecule import QMolecule
+from ....drivers import FermionicDriver
+from ....operators import FermionicOp
+from ....operators.second_quantization import SecondQuantizedOp
+from .integrals_calculators.angular_momentum_integrals_calculator import \
     calc_total_ang_momentum_ints
-from qiskit_nature.problems.second_quantization.molecular.fermionic_op_builder import \
-    build_fermionic_op, \
-    build_ferm_op_from_ints
-from qiskit_nature.problems.second_quantization.molecular.integrals_calculators \
-    .magnetization_integrals_calculator \
-    import calc_total_magnetization_ints
-from qiskit_nature.problems.second_quantization.molecular.integrals_calculators \
-    .particle_number_integrals_calculator \
-    import \
-    calc_total_particle_num_ints
-from qiskit_nature.transformations.second_quantization import BaseTransformer
+from ....problems.second_quantization.molecular.fermionic_op_builder import \
+    build_fermionic_op, build_ferm_op_from_ints
+from .integrals_calculators.magnetization_integrals_calculator import calc_total_magnetization_ints
+from .integrals_calculators.particle_number_integrals_calculator import calc_total_particle_num_ints
+from ....transformations.second_quantization import BaseTransformer
```

Comment:
```suggestion
from .fermionic_op_builder import build_fermionic_op, build_ferm_op_from_ints
from .integrals_calculators.magnetization_integrals_calculator import calc_total_magnetization_ints
from .integrals_calculators.particle_number_integrals_calculator import calc_total_particle_num_ints
from qiskit_nature.transformations.second_quantization import BaseTransformer
```

And here, the first one can be simplified a lot whereas the last one should again be absolute imo.